frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/LinkedReleasesAndProjectsAwarePortlet.java

```diff
@@ -107,6 +110,7 @@ protected void putDirectlyLinkedReleaseRelationsInRequest(PortletRequest request
 
     protected void putDirectlyLinkedReleasesInRequest(PortletRequest request, Project project) throws TException {
         List<ReleaseLink> linkedReleases = SW360Utils.getLinkedReleases(project, thriftClients, log);
+        Collections.sort(linkedReleases, compareByReleaseName(true));
```

Comment:
I would suggest use stream sorting,
```
linkedReleases.stream().sorted(Comparator.comparing(
    rl -> SW360Utils.getVersionedName(nullToEmptyString(rl.getName()), rl.getVersion()), String.CASE_INSENSITIVE_ORDER)
    ).collect(Collectors.toList());
```
rather than `Collections.sort()`.

Reply:
Changes done